### ISSUE TITLE
Three dots with flatpak app

### ISSUE THREAD

**Issue by plumlis** [ubuntu/gnome-shell-extension-appindicator#471]
Flatpak application ：https://flathub.org/apps/tech.feliciano.pocket-casts

tray icon displayed in three dots.

![Screenshot from 2023-09-27 16-56-28](https://github.com/ubuntu/gnome-shell-extension-appindicator/assets/6737429/cfa63486-b22e-49ba-b6ff-2dba559c689c)

Here are logs:

Sep 27 16:53:23 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:33 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:40 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:40 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:40 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:52 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:52 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:52 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:57 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:53:57 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:54:04 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:54:07 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb
Sep 27 16:54:24 plum-laptop appindicatorsupport@rgcjonas.gmail.com[2189]: pocket-casts1, Impossible to lookup icon for 'pocket-casts1_1' in /tmp/.org.chromium.Chromium.7DmNDb

**Comment by a73s** [comment]
I dont think this has anything to do with flatpak. For me it works with some things and not others, flatpak steam and keepass xc works, but flatpak qbittorrent doesn't. It also doesn't work with native nextcloud client app.

**Comment by xalt7x** [comment]
For me this app shows icon
![image](https://github.com/ubuntu/gnome-shell-extension-appindicator/assets/44649402/4148f8d8-07c6-4563-b90d-9133f69cb0ab)
Guess this issue has something to do with the screen resolution.
Have you tried changing "Icon size" in the extension preferences?

![image](https://github.com/ubuntu/gnome-shell-extension-appindicator/assets/44649402/f2522a2c-92df-4d4b-ac56-802ec7fc8911)

**Comment by kanansnote** [comment]
I have the same issue, as I have no idea how to fix that, particularly. 

By the way, looking at your screenshot, the font seems eye-catchy, Is it, by any chance, the Poppins font? I'd appreciate it if you could share the name with me. Thanks.

**Comment by xalt7x** [comment]
> By the way, looking at your screenshot, the font seems eye-catchy, Is it, by any chance, the Poppins font? I'd appreciate it if you could share the name with me.

It's Cantarell.

**Comment by kanansnote** [comment]
@xalt7x Are you sure it is? I am asking @plumlis, not you, actually.

**Comment by plumlis** [comment]
> @plumlis I have the same issue, as I have no idea how to fix that, particularly.
> 
> By the way, looking at your screenshot, the font seems eye-catchy, Is it, by any chance, the Poppins font? I'd appreciate it if you could share the name with me. Thanks.

This issue is flatpak packaging related maybe. Because some apps with this issue it fixed by update to new version(Flatpak)

The font name is harmonyos sans from huawei. You can download from here :
https://developer.harmonyos.com/en/docs/design/des-guides/font-0000001157868583

**Comment by RalfJung** [comment]
> It also doesn't work with native nextcloud client app.

Yeah, same problem here. And there's nothing in the logs about this that I can see.
How do I even find out which icon the extension is looking for and why it can't find it?

**Comment by haluzpav** [comment]
Same issue. For me the app `com.nextcloud.desktopclient.nextcloud` is giving me 3 dots. Also Jetbrains Toolbox but that's not flatpak. 

It's strange. Turning the extension off/on fixes the Toolbox (for some time?) but Nextcloud. Some other apps, for example Steam (not flatpak) or Cryptomator (flatpak) work fine. Playing with extension settings doesn't help.

Fedora 38 with X11.

**Comment by madhums** [comment]
Same issue with nextcloud-desktop-client on fedora 39.

**Comment by madhums** [comment]
It seems to work fine when installed from rpm instead of flathub.

**Comment by migtorr** [comment]
I have the same problem when I installed megasync rpm from toolbox (fedora silverblue), but when the rpm is installed from rpm-ostree there is a tray icon
